peps/pep-0770.rst

```diff
@@ -179,98 +204,62 @@ Specification
 
 The changes necessary to implement this PEP include:
 
-* Additions to `Core Metadata <770-spec-core-metadata_>`_, as defined in the
-  `Core Metadata specification <coremetadataspec_>`__.
-* Additions to the author-provided
-  `project source metadata <770-spec-project-source-metadata_>`_, as defined in the
-  `pyproject.toml specification <pyprojecttoml_>`__.
-* `Additions <770-spec-project-formats_>`_ to the source distribution (sdist),
-  built distribution (wheel), and installed project specifications.
+* A new reserved registry of subdirectory names in the ``.dist-info`` directory.
+* A new reserved optional ``[additional-files]`` table with an optional
+  ``sboms`` key added to
+  `project source metadata <770-spec-project-source-metadata_>`_,
```

Comment:
All top-level names in project source metadata are reserved. 
```suggestion
* An optional top-level table, ``[additional-files]``, added to
  `project source metadata <770-spec-project-source-metadata_>`_,
* An optional ``sboms`` key in the new ``[additional-files]`` table,
```

Reply:
Done in https://github.com/python/peps/pull/4283/commits/f0258e8e81db992953ee2424c7af1600b34070cb